cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -381,9 +448,9 @@ void ImageFlip::pcCallback(const sensor_msgs::PointCloud2::ConstPtr& point_cloud
 		for (int v = 0; v < point_cloud_out_msg->height; v++)
 			for (int u = 0; u < point_cloud_out_msg->width; u++)
 				memcpy(&(point_cloud_out_msg->data[v*row_size_out+u*element_size]), &(point_cloud_msg->data[(point_cloud_msg->height-1-u)*row_size+v*element_size]), element_size);
-		rot_mat.at<double>(0,1) = -1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.